libraries/platform/iot_network.h

```diff
@@ -48,6 +48,19 @@ typedef enum IotNetworkError
     IOT_NETWORK_SYSTEM_ERROR   /**< An error occurred when calling a system API. */
 } IotNetworkError_t;
 
+/**
+ * @ingroup platform_datatypes_enums
+ * @brief Disconnect reasons for [the network close callback](@ref platform_network_function_closecallback).
+ */
+typedef enum IotNetworkCloseReason
+{
+    IOT_NETWORK_NOT_CLOSED = 0,    /**< Not closed, still open */
```

Comment:
IOT_NETWORK_NOT_CLOSED :  I'm assuming this is the status of client initiated close ( to report close failure).  It does not look like the reason for the close of the connection. ( The enum is called "IotNetworkCloseReason_t"). So should this be  part of the reason?

Reply:
I had a hard time naming this one.  I was thinking we needed a reason code which represented "not closed" in the same way that a function can return a 0 value which means success.  Once we finish implementing the ports such that they can actually populate the reason code, we can investigate removing this value if it is never needed.